README.md

```diff
@@ -1,3 +1,13 @@
 # Tic Tac Toe in Clojure
 
-A Tic Tac Toe application written using Clojure.
+A program to play Tic Tac Toe, written in Clojure.
+
+## Instructions
```

Comment:
This is a good start.  There are parts that are probably a little too obvious to include here though.  Step 1, for example, could be left out.  The main thing that I see missing is developer instructions.  How do I run the tests?  How do I build, package, and run the final product?

Also, it would be nice to include some of your goals and motivations for the project.  This can include anything from personal goals like "learning functional programming" to features like "computer player is unbeatable" to technical details like "computer player uses a minimax algorithm".

Feel free to look through my Github projects and other Github projects to get an idea about how people write READMEs.